src/SingleMovie.js

```diff
@@ -56,7 +58,8 @@ class SingleMovie extends Component {
     return (
       <div className='single-movie-container'>
         <button className='back-button' onClick={this.props.focusViewOff}>Return to main</button>
-        {!this.state.movie ? <p>Sorry, please try again later.</p> : movieDetails}
+        {this.state.movie && movieDetails}
+        {this.state.error && <p className="error-message" >Sorry, something went wrong.  Please try again later.</p>}
```

Comment:
Depending on how useful the error message from a sad fetch is, we could actually use it here instead of hardcoding a message - something to think about 😄